README.md

```diff
@@ -68,7 +91,7 @@ and pass in the controller and the layout.
 
 ```java
 // ...
-@InjectController(controller = UserController.class, layout = R.layout.element_user)
+@RCModel(controller = UserListRecyclerController.class)
```

Comment:
should be UserController  or the controller should of be named UserListRecyclerController below 👀